### ISSUE TITLE
Publish to AltStore with each release

### ISSUE THREAD

**Issue by silentsilas** [buresdv/Mlem#166]
## Introduction
Currently, the TestFlight beta is full. Technically-inclined users would be able to work around this by side-loading the IPA via AltStore. 

## Requirements
* Create an IPA via Xcode
* Host it somewhere, lots of projects automate it with Github Releases
* Follow this [guide](https://faq.altstore.io/sources/make-a-source) to create a "source" that users can paste into AltStore to install the app

## Discussion
Alternatively, simply hosting an IPA file somewhere would suffice. AltStore lets you directly sideload an IPA from your PC/Mac to your connected device.

## What I Need
Essentially, the X Y problem is the need for an alternative method to install the app while the Testflight beta is full.

**Comment by Demon-tk** [comment]
Seconded

**Comment by norahqueen** [comment]
Agreed, would also like an IPA I can manually install through AltStore. I don't have the beta but I'm looking forward to adding my own sources.

**Comment by C0Nd3Mnd** [comment]
Would it be possible/allowed by Apple to release forks of this repository on TestFlight? If yes, people with paid developer accounts could release their "own" version on TestFlight for another 10,000 slots.

**Comment by MrAusnadian** [comment]
Can likely use https://github.com/marketplace/actions/ios-build-action for this
